sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update \
+operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="1991-1994 Xerox Corporation
+	1996-1999 Silicon Graphics
+	2003-2011 Hewlett-Packard Development Company, L.P.
+	2005,2007 Thiemo Seufer
+	2007 NEC LE-IT
+	2016 Marek Vasut"
+LICENSE="MIT
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ivmai/libatomic_ops/releases/download/v$portVersion/libatomic_ops-$portVersion.tar.gz"
+CHECKSUM_SHA256="219724edad3d580d4d37b22e1d7cb52f0006d282d26a9b8681b560a625142ee6"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libopsVersion="1.1.1"
+libgplVersion="1.1.2"
+
+libopsVersionCompat="$libopsVersion compat >= ${libopsVersion%%.*}"
+libgplVersionCompat="$libgplVersion compat >= ${libgplVersion%%.*}"
+
+PROVIDES="
+	libatomic_ops$secondaryArchSuffix = $portVersion
+	lib:libatomic_ops$secondaryArchSuffix = $libopsVersionCompat
+	lib:libatomic_ops_gpl$secondaryArchSuffix = $libgplVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libatomic_ops$secondaryArchSuffix_devel = $portVersion
+	devel:libatomic_ops$secondaryArchSuffix = $libopsVersionCompat
+	devel:libatomic_ops_gpl$secondaryArchSuffix = $libgplVersionCompat
+	"
+REQUIRES_devel="
+	libatomic_ops$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gawk
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	runConfigure ./configure --disable-dependency-tracking \
```

Comment:
Looks like this port make use of autoconf. Could you check if the library was built with debuginfo? If so we need a debuginfo package as @fbrosson [suggested](https://github.com/haikuports/haikuports/pull/2213#discussion_r168295424)